saleor/dashboard/order/forms.py

```diff
@@ -614,6 +614,16 @@ def __init__(self, *args, **kwargs):
         for form in self.forms:
             form.empty_permitted = False
 
+    def clean(self):
+        if any(self.errors):
```

Comment:
Why do we need that check? It's rather uncommon in clean methods and in our codebase as well I think. I've tested this code and if we removed it and below used `form.cleaned_data.get('quantity', 0)`, all errors would be returned properly.

Reply:
It's just for bypassing all following validations in advance if there are errors already, which makes the clean method faster. I will remove it if it's not suitable for saleor codebase.